### PR TITLE
Add Alibaba problem lists

### LINKED ISSUE #3731: MemoryTestZGC.sh fail by dragonwell11, should this testcase excluded when vendor is dragonwell
**Describe the bug**
test/jdk/java/lang/management/MemoryMXBean/MemoryTestZGC.sh fail by dragonwell11

```
STDOUT:
runOne -XX:+UnlockExperimentalVMOptions -XX:+UseZGC MemoryTest 1 1
STDERR:
Exception in thread "main" java.lang.RuntimeException: TEST FAILED: Number of memory managers = 4 but expected = 3
        at MemoryTest.checkMemoryManagers(MemoryTest.java:171)
        at MemoryTest.main(MemoryTest.java:100)
```

**To Reproduce**
```bash
jtreg test/jdk/java/lang/management/MemoryMXBean/MemoryTestZGC.sh
```

The WhiteBox API ManagementFactory.getMemoryManagerMXBeans() return by dragonwell11:
image

The WhiteBox API ManagementFactory.getMemoryManagerMXBeans() return by temurin11:
image

**Expected behavior**
I think there is some diffrence between dragonwell11 and temurin return from ManagementFactory.getMemoryManagerMXBeans(), but it shoule not treat a bug.

**Screenshots**

**Additional context**
temurin11 version:
```
openjdk version "11.0.15" 2022-04-19
OpenJDK Runtime Environment Temurin-11.0.15+10 (build 11.0.15+10)
OpenJDK 64-Bit Server VM Temurin-11.0.15+10 (build 11.0.15+10, mixed mode)
```

draonwell11 version:
```
openjdk version "11.0.15.11" 2022-04-19
OpenJDK Runtime Environment (Alibaba Dragonwell)-11.0.15.11+9-GA (build 11.0.15.11+9)
OpenJDK 64-Bit Server VM (Alibaba Dragonwell)-11.0.15.11+9-GA (build 11.0.15.11+9, mixed mode)
```

### PR DESCRIPTION
**Pull request by sophia-guo**
Fix #3731 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by llxia** [comment]
Should we also add alibaba in openjdk.mk file? https://github.com/adoptium/aqa-tests/blob/master/openjdk/openjdk.mk#L178

**Review by llxia — APPROVED** [review]
LGTM

File: openjdk/excludes/vendors/alibaba/ProblemList_openjdk11.txt [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+############################################################################
+
+# jdk_compiler
+
+############################################################################
+
+
+# jdk_lang
+java/lang/management/MemoryMXBean/MemoryTestAllGC.sh https://github.com/adoptium/aqa-tests/issues/3731 generic-all
```
Comment (sendaoYan) [inline openjdk/excludes/vendors/alibaba/ProblemList_openjdk11.txt:41]: I have made a mistake for the testcase name, the fail testcase is MemoryTestZGC.sh but not MemoryTestAllGC.sh

Comment (karianna) [reply openjdk/excludes/vendors/alibaba/ProblemList_openjdk11.txt:41]:
```suggestion
java/lang/management/MemoryMXBean/MemoryTestZGC.sh https://github.com/adoptium/aqa-tests/issues/3731 generic-all
```

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**